app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java

```diff
@@ -898,36 +912,14 @@ public void onError(Exception exception) {
         public void run() {
             if (!isPlaying()) return; // in case we completed since this was requested
 
+            // TODO move this into skip() if we are direct playing
```

Comment:
This todo is new, should it be solved now?

Reply:
Completely forgot about that. Latest push has it implemented.